docs/openapiv2/apidocs.swagger.json

```diff
@@ -32,7 +32,7 @@
     "/stores": {
       "get": {
         "summary": "List all stores",
-        "description": "Returns a paginated list of OpenFGA stores.",
+        "description": "Returns a paginated list of OpenFGA stores and a continuation token to get additional stores.\nThe continuation token will be empty if there are no more stores.\n",
```

Comment:
These comments should also be applied to the protobufs, otherwise the gRPC documentation stubs won't have this info as well.

Reply:
Added to protobufs

Reply:
This is a larger effort to be track via https://github.com/openfga/api/issues/84